src/tools/dart-tool.md

```diff
@@ -47,28 +47,29 @@ The following table shows which commands you can use with the `dart` tool.
 |---------+--------------------------------+-----------------------------------|
 | Command | Example of use                 | More information                  |
 |---------|--------------------------------|-----------------------------------|
-| analyze | `dart analyze`     | Analyzes the project's Dart source code.<br>Use instead of [`dartanalyzer`][]. |
-| compile | `dart compile exe <DART_FILE>` | Compiles Dart to various formats.<br>Use instead of [`dart2js`][] and [`dart2native`][]. | 
-| create  | `dart create <DIRECTORY>`      | Creates a new project.<br>Use instead of [`stagehand`.][] | 
-| format  | `dart format <DIRECTORY|DART_FILE>` | Formats Dart source code.<br>Use instead of [`dartfmt`][]. | 
-| pub     | `dart pub <PUB_COMMAND>`       | Works with packages.<br>Use instead of [`pub`][]. | 
-| run     | `dart run <DART_FILE>`         | Runs a Dart program. <br>Use instead of the pre-existing [Dart VM command][dart-vm]. | 
-| test    | `dart test [<DIRECTORY|DART_FILE>]` | Runs tests in this package.<br>Use instead of [`pub run test`][`pub`]. |
+| `analyze` | `dart analyze`     | Analyzes the project's Dart source code.<br>Use instead of `dartanalyzer`.<br>[Learn more.][analyze] |
+| `compile` | `dart compile exe <DART_FILE>` | Compiles Dart to various formats.<br>Use instead of `dart2js` and `dart2native`.<br>[Learn more.][compile] | 
+| `create`  | `dart create <DIRECTORY>`      | Creates a new project.<br>Use instead of [`stagehand`.][] | 
+| `fix`  | `dart fix <DIRECTORY|DART_FILE>` | Migrates Dart code to use newer syntax styles.<br>Use instead of [`dartfix`][]. | 
```

Comment:
Instead of `Migrates Dart code to use newer syntax styles.`, perhaps:

```
Fixes analysis issues and performs automated API migrations.
```

Also, perhaps change `Use instead of dartfix` to `Use instead of the deprecated dartfix`?

Reply:
Or perhaps `Apply automated fixes to Dart source code` (which is the tools help output).

Reply:
I like the description. Adding "the deprecated" seems unnecessary, though — "instead of" implies that for me, and I'd like to avoid cluttering the table.